src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -69,7 +68,7 @@ public virtual Task<bool> ShouldHandleRequestAsync()
                 {
                     return true;
                 }
-                else if (authResult.Nothing)
+                else if (authResult.Skipped || authResult.Nothing)
```

Comment:
When I rename Nothing today, I'm going to take another look to see if we can simplify this into a single check again